protocol/avdl/config.avdl

```diff
@@ -13,6 +13,21 @@ protocol config {
 
   GetCurrentStatusRes getCurrentStatus(int sessionID);
 
+  record ExtendedStatus {
+    boolean standalone;
+    boolean passphraseStreamCached;
+    DeviceID deviceID;
+    string deviceName;
+    string deviceStatus;
+    string logDir;
+    boolean desktopUIConnected;
```

Comment:
eventually we should also show: pid, path, and process name/type; not just if is connected

It's slightly weird `ExtendedStatus` isn't a superset of `Status`.  The overlap is actually quite small.